src/setup-tests.ts

```diff
@@ -0,0 +1,43 @@
+import { act } from '@testing-library/react'
+import type { State, StateCreator, StoreMutatorIdentifier } from './vanilla'
+import actualCreate from './'
+
+// maps use more memory than sets, but we can use them for
```

Comment:
I think you can use a `Set`.
Not from the memory perspective, but I feel like `uid` is unnecessary.